src/euroeval/benchmark_modules/litellm.py

```diff
@@ -219,24 +220,36 @@ def generative_type(self) -> GenerativeType | None:
         Returns:
             The generative type of the model, or None if it has not been set yet.
         """
+        param = ""
+
+        if self.model_config.parameter:
+            param = self.model_config.parameter
+        elif self.model_config.revision in ALLOWED_PARAMS.get(self.model_config.model_id, []):
+            import warnings
+            warnings.warn(
+        f"Using '@' for parameters (like '{self.model_config.revision}') is deprecated. "
+        f"Please use '#' instead (e.g., '{self.model_config.model_id}#{self.model_config.revision}').",
+        DeprecationWarning,
+        stacklevel=2
+    )
+            param = self.model_config.revision
```

Comment:
Nit: Adding an extra newline to separate this `param` block from the others.
```suggestion
            param = self.model_config.revision
            
```